lib/src/mediator_client/options/web_socket_options.dart

```diff
@@ -16,6 +16,12 @@ class WebSocketOptions {
   /// WebSocket connection alive.
   final int pingIntervalInSeconds;
 
+  /// Indicates whether the message should be deleted on the mediator right after it was received by the client (default: true).
+  final bool deleteOnMediator;
```

Comment:
Wdyt about:

```suggestion
  /// Messages are deleted from the mediator as soon as they are received by the client. (default: true)
  ///
  /// NOTE: This may lead to the client application loosing messages if it crashes while processing a message. For high durability it is recommended to set `deleteOnConsume` to false and explicitly delete messages once processed.
  final bool deleteOnConsume;
```